cli/pkg/kctrl/local/reconciler.go

```diff
@@ -213,6 +216,11 @@ func (o *Reconciler) newReconcilers(
 	refTracker := reftracker.NewAppRefTracker()
 	updateStatusTracker := reftracker.NewAppUpdateStatus()
 
+	kubeConfig := kubeconfig.NewKubeconfig(coreClient, runLog)
+	compInfo := componentinfo.NewComponentInfo(coreClient, kubeConfig, "dev")
+
```

Comment:
nit: remove new line

Reply:
I kept it on purpose :)
If you take a look at the above declarations above, they are also grouped in a similar fashion.

Reply:
Ah gotchya!